src/ProjectAudioManager.cpp

```diff
@@ -904,6 +904,7 @@ bool ProjectAudioManager::DoRecord(AudacityProject &project,
                   t1 = newTrack->LongSamplesToTime(newTrack->TimeToLongSamples(t1));
             }
 
+            auto tempList = TrackList::Temporary(nullptr, newTrack, nullptr);
```

Comment:
I've tried to say during the standup "Oh, wait, I saw the fix in a different PR", but I was muted.

So...

Oh, wait, I saw the fix in a different PR :-)